tests/console/sshd.pm

```diff
@@ -135,7 +136,6 @@ sub run {
 
     assert_script_run "killall -u $ssh_testman || true";
     wait_still_screen 3;
-    clear_console;
```

Comment:
I assume this was added so that the `job killed` messages are cleared. If there is a needle matching the prompt, it might not match at this point anymore.

Reply:
change to `clear_console if !is_serial_terminal;`